src/visualizers/visualizerSpace.js

```diff
@@ -2,12 +2,11 @@
 /**
  * Created by Michael on 31/12/13.
  * modified by henryz00
- * original repo: https://github.com/michaelbromley/soundcloud-visualizer
+ * original repo: https://github.canalyserom/michaelbromley/soundcloud-visualizer
  */
 let visualizer;
 let audioSource;
-let analyser;
-let dataArray;
+let audioData;
 let container;
 
 const PlayerAudioSource = function (player) {
```

Comment:
This seems like it ought to become several `class`es.

Reply:
What do you mean?

Reply:
You see `PlayAudioSource` is associated with the `function (player) { ... }` right? This is an old ES5 way to construct objects. In ES6/ES2015+, we should always favor `class` constructs instead.

Reply:
Right, indeed